modules/client/test/integration/client/methods.test.ts

```diff
@@ -504,126 +529,126 @@ This code creates a dao with the legacy method
-      name: "TestToken" +
-        (Random.getFloat() + 1)
-          .toString(36)
-          .substring(4)
-          .toUpperCase(),
-      symbol: "TEST" +
-        (Random.getFloat() + 1)
-          .toString(36)
-          .substring(4)
-          .toUpperCase(),
-    },
-    mintConfig: {
-      receivers: [Wallet.createRandom().address, Wallet.createRandom().address],
-      amounts: [
-        BigInt(Math.floor(Random.getFloat() * 9999) + 1),
-        BigInt(Math.floor(Random.getFloat() * 9999) + 1),
-      ],
-    },
-    votingConfig: {
-      supportRequiredPct: Math.floor(Random.getFloat() * 100) + 1,
-      participationRequiredPct: Math.floor(Random.getFloat() * 100) + 1,
-      minDuration: Math.floor(Random.getFloat() * 9999) + 1,
-    },
-    // gsnForwarder: Wallet.createRandom().address,
-  };
-  const registryInstance = await daoFactoryInstance
-    .registry()
-    .then((registryAddress) => {
-      return Registry__factory.connect(registryAddress, provider[0]);
-    });
-  return daoFactoryInstance.newERC20VotingDAO(
-    daoCreationParams.daoConfig,
-    daoCreationParams.votingConfig,
-    daoCreationParams.tokenConfig,
-    daoCreationParams.mintConfig,
-    daoCreationParams.gsnForwarder ?? AddressZero,
-  )
-    .then((tx) => tx.wait())
-    .then((cr) => {
-      const newDaoAddress = cr.events?.find(
-        (e) => e.address === registryInstance.address,
-      )?.topics[1];
-      if (!newDaoAddress) {
-        return Promise.reject(new Error("Could not create DAO"));
-      }
-
-      return "0x" + newDaoAddress.slice(newDaoAddress.length - 40);
-    });
-}
-
-interface ICreateDaoERC20Voting {
-  daoConfig: DaoConfig;
-  tokenConfig: TokenConfig;
-  mintConfig: MintConfig;
-  votingConfig: VotingConfig;
-  gsnForwarder?: string;
-}
-
-interface DaoConfig {
-  name: string;
-  metadata: string;
-}
-
-interface TokenConfig {
-  addr: string;
-  name: string;
-  symbol: string;
-}
-
-interface MintConfig {
-  receivers: string[];
-  amounts: bigint[];
-}
-
-interface VotingConfig {
-  /** 0-100 as a percentage */
-  supportRequiredPct: number;
-  /** 0-100 as a percentage */
-  participationRequiredPct: number;
-  /** In seconds */
-  minDuration: number;
-}
-
-function useWeb3Providers(
-  endpoints: string | JsonRpcProvider | (string | JsonRpcProvider)[],
-  network: Networkish,
-): JsonRpcProvider[] {
-  if (Array.isArray(endpoints)) {
-    return endpoints.map((item) => {
-      if (typeof item === "string") {
-        const url = new URL(item);
-        return new JsonRpcProvider(url.href, network);
-      }
-      return item;
-    });
-  } else if (typeof endpoints === "string") {
-    const url = new URL(endpoints);
-    return [new JsonRpcProvider(url.href, network)];
-  } else {
-    return [endpoints];
-  }
-}
+// async function createLegacyDao(params: ContextParams) {
+//   if (!params.web3Providers) {
+//     throw new Error("A provider is needed");
+//   }
+//   if (!params.daoFactoryAddress) {
```

Comment:
Will we need this legacy function in the future? If not, I'd just delete the commented code

Reply:
deleted